sharding-core/sharding-core-parse/sharding-core-parse-mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -20,27 +20,45 @@ grammar BaseRule;
 import Keyword, DataType, Symbol;
 
 ID 
-    : (BQ_?[a-zA-Z_$][a-zA-Z0-9_$]* BQ_? DOT_)? (BQ_?[a-zA-Z_$][a-zA-Z0-9_$]* BQ_?)
+    : [A-Za-z_$0-9]*?[A-Za-z_$]+?[A-Za-z_$0-9]*
+    ;
+    
+REVERSE_QUOTE_ID
+    : '`' ~'`'+ '`'
+    ;
+    
+uid
+    : ID
+    | REVERSE_QUOTE_ID
+    | keywordCanBeID
+    ;
+    
+keywordCanBeID
+    : DATE | PASSWORD
+    ;
+    
+fullID
```

Comment:
what is full id? qualified name?

Reply:
full id may be with schema name database name.

Reply:
Try to find terminology word to instead of this, check `qualified name` is suitable or not